enterprise/frontend/src/embedding-sdk/package.template.json

```diff
@@ -17,7 +17,8 @@
       "types": "./dist/nextjs.d.ts",
       "require": "./dist/nextjs.cjs",
       "import": "./dist/nextjs.js"
-    }
+    },
+    "./package.json": "./package.json"
```

Comment:
Will this make it show up in the autocomplete when importing from the host app?

Does `resolve(`@metabase/embedding-sdk-react`) resolve to the root folder? if so I think we can just manually calculate the path to the package.json from that maybe?

Reply:
@npretto resolve was resolving to "index.ts" in the source folder (enterprise/frontend/src/embedding-sdk), weirdly enough, so in https://github.com/metabase/metabase/pull/51268 I manually resolved to node_modules in the component tests.

Reply:
> Will this make it show up in the autocomplete when importing from the host app?

Good question, let me try to build this locally and see if it affects imports.